app/src/main/java/org/wikipedia/feed/suggestededits/SuggestedEditsFeedClient.kt

```diff
@@ -33,6 +35,7 @@ class SuggestedEditsFeedClient(private var invokeSource: Constants.InvokeSource)
 
     override fun request(context: Context, wiki: WikiSite, age: Int, cb: FeedClient.Callback) {
         this.age = age
+        this.context = context
```

Comment:
Can we not store the instance of `Context`, and instead use `WikipediaApp.getInstance()`?

Reply:
Yes, done.